Big-Spender/readme.md

```diff
@@ -68,39 +68,39 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where lower (description) like '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends s inner join expense_areas ea on (s.expense_area_id = ea.id) WHERE expense_area = 'Better Hospital Food';
```

Comment:
with * is this all spends? all expense_areas? both? 

Is there a difference between join and inner join?